docs/_sidebar.md

```diff
@@ -14,8 +14,15 @@
   - [State Diagram](stateDiagram.md)
   - [Gantt](gantt.md)
   - [Pie Chart](pie.md)
+
 - Guide
 
   - [Development](development.md)
   - [mermaidAPI](mermaidAPI.md)
-  - [Changelog](CHANGELOG.md)
\ No newline at end of file
+  - [Changelog](CHANGELOG.md)
+
+- I'm a n00b
```

Comment:
I wouldn't make a hard cut between beginners and intermediates here. Maybe the parts for beginners can be integrated into the regular docs structure.